test/inlining/rlexe.xml

```diff
@@ -139,6 +139,7 @@
       <files>bug515849.js</files>
       <baseline>bug515849.baseline</baseline>
       <compile-flags>-minInterpretCount:1 -maxInterpretCount:1 -msjrc:0 -force:inline</compile-flags>
+      <tags>exclude_arm64</tags>
```

Comment:
>exclude_arm64 [](start = 12, length = 13)

Why? if this is failing due to slowness/bug add a comment to re-enable later

Reply:
The test is somewhat recursive and on arm64 it's hitting "Out of stack space" before it reaches it's normal completion.